file.c

```diff
@@ -7135,7 +7135,7 @@ const char ruby_null_device[] =
  *
  *  Note that file permissions are quite different from the _mode_
  *  of a file stream (\File object).
- *  See IO@Modes.
+ *  See IO.
```

Comment:
Is there a specific part we can link to rather than the generic IO page?

Reply:
Sorry, I thought I'd responded here.  I looked, but didn't see as suitable target.

Reply:
Can you fix this one in a separate PR then? (Or alternatively delete this completely). I don't think asking the reader to read all of the docs of IO is particularly helpful.

Reply:
Sure.  I see the problem, but not the solution.